### PR TITLE
added sample html response

### PR DESCRIPTION
**Pull request by alignatiev07**

### CODE REVIEW

File: service/src/main/scala/kafka/console/service/Application.scala [hunk 1]
```diff
@@ -25,6 +26,12 @@ object Application {
        topicService andThen getTopics
   }
 
+  private val html = raw {
+    case GET -> Root / "html" =>
+      Ok(scalaz.stream.io.linesR(getClass.getClassLoader.getResource("webpage.html").getPath))
```
Comment (eugene-cheverda) [inline service/src/main/scala/kafka/console/service/Application.scala]: getClass.getClassLoader.getResource("webpage.html") can be shortened to getClass.getResource("/webpage.html")

Comment (eugene-cheverda) [inline service/src/main/scala/kafka/console/service/Application.scala]: scalaz.stream.io could be imported

File: service/src/main/resources/webpage.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <title>Task Element</title>
+  </head>
+  <body>
+    <table width="100%" border="0" cellspacing="0" cellpadding="2" class="NavBar">
+      <tr>
+        <td class="NavBar-Cell">
+          <a href="http://nant.sourceforge.net">
+            <b>NAnt</b>
+          </a>
+          <img alt="-&gt;" src="../images/arrow.gif" />
```
Comment (eugene-cheverda) [inline service/src/main/resources/webpage.html]: There are no additional resources in this PR. The proposal is either simplify webpage to be just static text or add images, csses and other resources as well.

File: service/src/main/scala/kafka/console/service/Application.scala (hunk 1 above)
Comment (alignatiev07) [inline service/src/main/scala/kafka/console/service/Application.scala]: It seems to me that I was missing a slash ('/'). That's why shortened version didn't work for me while I was testing

**Pull request merged**